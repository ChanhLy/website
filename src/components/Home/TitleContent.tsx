export interface TitleContentProps {
  className?: string;
}

export const TitleContent: React.FC<TitleContentProps> = (props) => {
  return (
    <div className="" {...props}>
      <div className="pt-16 pb-8">
        <div className="	 text-gray-900 flex justify-between">
          {/* {AppConfig.title} */}
          <div className="block text-center">
            <button className="rounded-full	bg-gray-300">IMG</button>
            <h1 className="font-bold my-5">Your title</h1>
            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
          </div>

          <div className="block text-center">
            <button className="rounded-full	bg-gray-300">IMG</button>
            <h1 className="font-bold my-5">Your title</h1>
            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
          </div>

          <div className="block text-center">
            <button className="rounded-full	bg-gray-300">IMG</button>
            <h1 className="font-bold my-5">Your title</h1>
            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
          </div>
        </div>
        {/* <div className="text-xl">{AppConfig.description}</div> */}
      </div>
      <div className="block text-center my-9">
        <h1 className="font-bold text-blue-500">Features</h1>
        <h1 className="font-bold  text-5xl my-5">Your title here</h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum
          dolor sit amet, consectetur adipiscing elit.
        </p>
      </div>
    </div>
  );
};
