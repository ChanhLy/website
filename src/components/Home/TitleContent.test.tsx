import React from 'react';

import renderer from 'react-test-renderer';

import { TitleContent } from './TitleContent';

it('renders correctly', () => {
  const tree = renderer.create(<TitleContent></TitleContent>).toJSON();
  expect(tree).toMatchSnapshot();
});
