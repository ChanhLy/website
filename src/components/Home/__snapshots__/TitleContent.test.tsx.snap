// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=""
>
  <div
    className="pt-16 pb-8"
  >
    <div
      className="	 text-gray-900 flex justify-between"
    >
      <div
        className="block text-center"
      >
        <button
          className="rounded-full	bg-gray-300"
        >
          IMG
        </button>
        <h1
          className="font-bold my-5"
        >
          Your title
        </h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </p>
      </div>
      <div
        className="block text-center"
      >
        <button
          className="rounded-full	bg-gray-300"
        >
          IMG
        </button>
        <h1
          className="font-bold my-5"
        >
          Your title
        </h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </p>
      </div>
      <div
        className="block text-center"
      >
        <button
          className="rounded-full	bg-gray-300"
        >
          IMG
        </button>
        <h1
          className="font-bold my-5"
        >
          Your title
        </h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </p>
      </div>
    </div>
  </div>
  <div
    className="block text-center my-9"
  >
    <h1
      className="font-bold text-blue-500"
    >
      Features
    </h1>
    <h1
      className="font-bold  text-5xl my-5"
    >
      Your title here
    </h1>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    </p>
  </div>
</div>
`;
